virttest/qemu_vm.py

```diff
@@ -3045,6 +3045,16 @@ def get_pid(self):
         except (TypeError, IndexError, ValueError):
             return None
 
+    def get_qemu_threads(self):
+        """
+        Return the list of qemu SPIDs
+
+        :return: the list of qemu SPIDs
+        """
+        cmd = "ls /proc/%d/task" % self.get_pid()
+        threads_list = commands.getoutput(cmd).split()
```

Comment:
if status not equal 0, output are error message,  output looks like:  
>>> z = commands.getoutput("kk")
>>> z
'sh: kk: command not found'
so, I think you should check status of command before return it.